library-front/src/components/Header/Header.css

```diff
@@ -60,3 +96,9 @@
 .header-right .register:hover {
   background-color: #41c8aa;
 }
+
+@media only screen and (min-width: 769px) {
+  .header.hidden {
+    transform: translateY(0);
```

Comment:
this is okay because it's probably position fixed element. In the future control element visibility by setting display to hide